package.json

```diff
@@ -100,6 +104,8 @@
     "es6-promise": "^3.0.2",
     "file-saver": "^1.3.3",
     "font-awesome": "^4.5.0",
+    "glsl-parser": "git+https://github.com/anvaka/glslx.git#glsl-parser",
```

Comment:
What's the reason for using this fork? :)

Reply:
This fork exposes the actual parser so that we can import it. The original repository doesn't do this unfortunately :/